subprojects/packagefiles/lmdb/meson.build

```diff
@@ -0,0 +1,29 @@
+project('lmdb', 'c', version: '0.9.29', license: 'OpenLDAP')
+
+cc = meson.get_compiler('c')
+
+src = [
+  'libraries/liblmdb/mdb.c',
+  'libraries/liblmdb/midl.c',
+]
+
+c_args = [
+  '-W',
+  '-Wall',
```

Comment:
What is this for?

Note that -Wall specifically should be set by project default_options specifying a meson `warning_level`.